README.md

```diff
@@ -1,11 +1,140 @@
-# lambda-template
-A GitHub template for quickly starting a new AWS lambda project.
+# cfn-macro-cost-rules
+A CloudFormation snippet macro for generating the Rules parameter of AWS::CE::CostCategory resources.
+
+## Input Format
+This snippet macro expects to process a snippet with one or both of two
+top-level keys: `InheritedValues` and `RegularValues`.
+
+### Inherited Values from Tags
+
+When specifying `InheritedValues`, a `TagOrder` sub-key must be provided, and a
+`RulePosition` sub-key may also be provided. The `TagOrder` sub-key must
+provide a list strings corresponding to a tag name that has been configured as a
+cost allocation tag. The `RulePosition` sub-key may be provided to configure
+the inherited-value rules to occur either before or after regular-valued rules;
+possible values are either `First` or `Last`, defaulting to `Last`.
+
+Minimal example creating a cost category from a single tag:
+```yaml
+Resources:
+  MyCostCategory:
+    Type: AWS::CE::CostCategory
+    Properties:
+      Name: 'My Cost Category'
+      RuleVersion: 'CostCategoryExpression.v1'
+      Rules:
+        Fn::Transform:
+      - Name: CostRulesGenerator
+    InheritedValues:
+      TagOrder:
+        - MyTagName
+```
+
+### Regular Values
```

Comment:
should it be possible to create a rule by passing both `InheritedValues` and `RegularValues`?  Is that something that cost explorer users should know?

Reply:
I've expanded on the README to add a section about combining them